Tests/AwesomeAssertions.Specs/Primitives/StringAssertionSpecs.EndWith.cs

```diff
@@ -46,7 +47,7 @@ public void When_string_does_not_end_with_expected_phrase_it_should_throw()
 
             // Assert
             act.Should().Throw<XunitException>().WithMessage(
-                "Expected string to end with \"AB\" because it should, but \"ABC\" differs near \"ABC\" (index 0).");
+                "Expected string*it should*index 2*ABC*AB*");
```

Comment:
Why using here the short version instead of full validation?
Maybe we need some kind of definition when to use the full validation!?